test/epicenter_web/live/profile_live_test.exs

```diff
@@ -1164,22 +1164,22 @@ defmodule EpicenterWeb.ProfileLiveTest do
 
     test "handles assign_users message when the changed people include the current person", %{person: alice, assignee: assignee, user: user} do
       billy = Test.Fixtures.person_attrs(assignee, "billy") |> Cases.create_person!()
-      socket = %Phoenix.LiveView.Socket{assigns: %{person: alice, current_user: user}}
+      socket = %Phoenix.LiveView.Socket{assigns: %{person: alice, current_user: user, __changed__: %{}}}
```

Comment:
Does this still pass if you use [assert/2](https://hexdocs.pm/ex_unit/1.13.0/ExUnit.Assertions.html#assert/2) with [match?/2](https://hexdocs.pm/elixir/Kernel.html#match?/2)?

Reply:
I don't follow. Which assertion would you like to see changed, and how?

Reply:
btw this is an example of being tightly coupled to the internal structure of `%Phoenix.LiveView.Socket`. Phoenix added this `__changed__` attribute, and change-tracking relies on it, so we need to init the socket with that attribute in the test.

I'll see if I can find a public interface that initializes the socket correctly rather than us constructing a struct like that.

Reply:
Interestingly, even the place that uses `configure_socket` [explicitly sets `__changed__`](https://github.com/phoenixframework/phoenix_live_view/blob/a038f3b9d5fbb2b4623f8a3d68dae6082c8c06bd/lib/phoenix_live_view/static.ex#L103) - so I'll live with it here.

Reply:
Ah, so is it the call to `ProfileLive.handle_info/2` that fails if you don't set `__changed__`?

Reply:
Yeah. It's the underlying call to `assign` inside of `handle_info` that relies on `__changed__` being present. This is a unit test, and as far as I can tell, Phoenix LiveView doesn't really promote this direct kind of unit testing.